### ISSUE TITLE
Suggest to use substring manipulation instead of cut

### ISSUE THREAD

**Issue by ColemanTom** [koalaman/shellcheck#2607]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
a=12345678
b=$(echo "$a" | cut -c 5-6)
export b

```

#### Here's what shellcheck currently says:

```

$ shellcheck myscript
No issues detected!

$ 

```

#### Here's what I wanted or expected to see:

Recommendation to use substring manipulation instead, e.g. `b=${a:4:2}`
